<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EditTagDefinition</class>
 <widget class="QDialog" name="EditTagDefinition">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>600</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>700</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>700</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Edit/Create Tag Definitions</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>681</width>
     <height>571</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="dialogGrid">
    <item row="3" column="2">
     <spacer name="hSpacer4">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="2" column="2" colspan="2">
     <spacer name="hSpacer3">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>358</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="2" column="1">
     <widget class="QPushButton" name="buttonRemoveTag">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>Remove Tag</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0" colspan="4">
     <widget class="QTableView" name="table">
      <property name="enabled">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QPushButton" name="buttonAddTag">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>Add Tag</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0" colspan="4">
     <layout class="QHBoxLayout" name="hLayout1">
      <item>
       <widget class="QLabel" name="label_groupName">
        <property name="text">
         <string>Tag Group Name:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="groupName">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>150</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>150</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="inputMask">
         <string/>
        </property>
        <property name="maxLength">
         <number>15</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_groupHelp">
        <property name="text">
         <string>(Max. 15 characters)</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="hSpacer1">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="buttonCreateRemove">
        <property name="text">
         <string>Create Group</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="3" column="3">
     <widget class="QPushButton" name="buttonDone">
      <property name="text">
       <string>Done</string>
      </property>
      <property name="default">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
